es6-classes/square.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(width) {
+    super(width);
+    this.area = width * width;
+    this.perimeter = width * 4;
```

Comment:
You should not be creating these properties manually. When you call `super()` it will generate these for you. So utilizing your `width` parameter, you need to calculate what `area` or `perimeter` would be, and that's what you pass into `super`